src/uct/cuda/cuda_copy/cuda_copy_md.c

```diff
@@ -594,6 +862,20 @@ uct_cuda_copy_md_open(uct_component_t *component, const char *md_name,
         md->config.dmabuf_supported = dmabuf_supported;
     }
 
+    if (config->enable_fabric != UCS_NO) {
+        md->config.fabric_supported = 1;
+    }
```

Comment:
we need to save the ternary value as-is on the md:
`md->config.fabric_supported = config->enable_fabric;`
to allow fallback to legacy pinned memory in case of try, and disable the fallback in case of yes